### PR TITLE
[Example] Add Keyword spotting example for GNA.

### PR DESCRIPTION
**Pull request by miaobin**
Add a keyword spotting model in our speech commands example. It only use 'FullyConnected' and  'RELU'. So it should be compatible with GNA.

PTAL @belem @BruceDai @fujunwei .
Thanks!

### CODE REVIEW

File: examples/util/base.js [hunk 1]
```diff
@@ -777,17 +777,32 @@ emotionAnalysisModels: [{
...
+    paperUrl: 'https://www.isca-speech.org/archive/interspeech_2015/papers/i15_1478.pdf'
+  }, {
+    modelName: 'KWS DNN (TFLite)',
+    format: 'TFLite',
+    modelId: 'kws_dnn_tflite',
+    modelSize: '320kB',
+    inputSize: [1, 250],
+    outputSize: [1, 12],
+    sampleRate: 16000,
+    modelFile: '../speech_commands/model/kws_dnn.tflite',
```
Comment (BruceDai) [inline examples/util/base.js]: @miaobin Please update [README.md](https://github.com/intel/webml-polyfill/blob/master/examples/speech_commands/model/README.md), add the link of this kws_dnn model and its descriptions, thanks

Comment (huningxin) [reply examples/util/base.js]: could we use OpenVINO models instead?

Comment (miaobin) [reply examples/util/base.js]: > could we use OpenVINO models instead?

Yes, the model have been converted to both tflite and OpenVINO IR format.

Comment (miaobin) [reply examples/util/base.js]: > @miaobin Please update [README.md](https://github.com/intel/webml-polyfill/blob/master/examples/speech_commands/model/README.md), add the link of this kws_dnn model and its descriptions, thanks

OK, I will update the README.

Comment (huningxin) [reply examples/util/base.js]: > Yes, the model have been converted to both tflite and OpenVINO IR format.

Great. Then could this example use the OpenVINO IR?

Comment (miaobin) [reply examples/util/base.js]: > > Yes, the model have been converted to both tflite and OpenVINO IR format.
> 
> Great. Then could this example use the OpenVINO IR?

OK, you think we'll use the both or just the OpenVINO IR?

Comment (huningxin) [reply examples/util/base.js]: > Great. Then could this example use the OpenVINO IR?

I would suggest OpenVINO IR first.

Comment (miaobin) [reply examples/util/base.js]: > > Great. Then could this example use the OpenVINO IR?
> 
> I would suggest OpenVINO IR first.

OK, I'll update this commit.

**Comment by miaobin** [comment]
- Use OpenVINO IR format model.
- Add steps to show how to convert model to OpenVINO IR format.

@belem @huningxin @BruceDai @fujunwei PTAL.
Thanks.

**Comment by fujunwei** [comment]
LGTM, it works on GNA plugin.

**Comment by ibelem** [comment]
Thanks @miaobin for the great work and thank @BruceDai and @fujunwei for the review.

**Pull request merged**